compose/local/django/start

```diff
@@ -6,4 +6,5 @@ set -o nounset
 
 
 python manage.py migrate
+python manage.py add_pipeline
```

Comment:
I'm glad you were able to write a management command! those are very useful. However, I have another place for generating the test data when the project starts up:
https://github.com/RolnickLab/antenna/blob/main/ami/tests/fixtures/main.py

You can see there are functions for create_occurrences, create_taxa, etc. So you can move create_pipeline there.

Then you can call that function from `setup_test_project` in that same file.

This is the function that sets up the entire test project and calls a few more of the fixture functions. 
https://github.com/RolnickLab/antenna/blob/fddcd76f7a8231a43c95f1a3d33373301edd7c85/ami/tests/fixtures/signals.py#L20